closure/repositories.bzl

```diff
@@ -726,23 +738,23 @@ def com_google_jsinterop_annotations():
 def com_google_protobuf():
     http_archive(
         name = "com_google_protobuf",
-        strip_prefix = "protobuf-3.9.0",
-        sha256 = "2ee9dcec820352671eb83e081295ba43f7a4157181dad549024d7070d079cf65",
+        strip_prefix = "protobuf-3.10.0-rc1",
+        sha256 = "690ea9b2ebaa788fbf2cf3353bba2e75bd99e16b09f5e69fdb664f679e5bc584",
         urls = [
-            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.9.0.tar.gz",
-            "https://github.com/protocolbuffers/protobuf/archive/v3.9.0.tar.gz",
+            #            "https://mirror.bazel.build/github.com/google/protobuf/archive/v3.10.0-rc1.tar.gz",
```

Comment:
The commented out mirror URL can be improved here and in other place.

@meteorcloudy @laurentlb Can you please upload `https://github.com/protocolbuffers/protobuf/archive/v3.10.0-rc1.tar.gz` to https://mirror.bazel.build/github.com/google/protobuf/archive/v3.10.0-rc1.tar.gz

Reply:
it seems like `3.10.0` was just released yesterday. maybe the `-rc1` suffix can be dropped?

Reply:
thank you for working on this btw!

Reply:
Thanks for letting me know, I will amend this series and bump right to 3.10.0.

@meteorcloudy @laurentlb Could you please upload https://github.com/protocolbuffers/protobuf/archive/v3.10.0.tar.gz to bazel mirror? Thank you in advance!

Reply:
Done, https://mirror.bazel.build/github.com/google/protobuf/archive/v3.10.0.tar.gz and https://mirror.bazel.build/github.com/google/protobuf/archive/v3.10.0.zip are both available.

Reply:
Thanks to everyone for help!

@gkdn @laurentlb @dslomov PTAL, it's ready now.